cli/deployments/registry.go

```diff
@@ -52,8 +54,32 @@ func (k Registry) Describe() string {
 	return emoji.Sprintf(":cloud:Registry version: %s\n:clipboard:Registry chart: %s", registryVersion, registryChartFile)
 }
 
+// Delete removes Registry from kubernetes cluster
 func (k Registry) Delete(c kubernetes.Cluster) error {
-	return c.Kubectl.CoreV1().Namespaces().Delete(context.Background(), registryDeploymentID, metav1.DeleteOptions{})
+	message := "Deleting Registry"
+	s := spinner.New(spinner.CharSets[11], 100*time.Millisecond)
+	s.Suffix = emoji.Sprintf(" %s :zzz:", message)
+	s.Start()
+
+	currentdir, err := os.Getwd()
+	if err != nil {
+		return errors.New("Failed uninstalling Registry: " + err.Error())
+	}
+
+	helmCmd := fmt.Sprintf("helm uninstall '%s' --namespace '%s'", registryDeploymentID, registryDeploymentID)
+	if out, err := helpers.RunProc(helmCmd, currentdir, k.Debug); err != nil {
+		return errors.New("Failed uninstalling Registry: " + out)
+	}
+
+	err = c.Kubectl.CoreV1().Namespaces().Delete(context.Background(), registryDeploymentID, metav1.DeleteOptions{})
+	if err != nil {
+		return errors.New("Failed uninstalling Registry: " + err.Error())
+	}
+	s.Stop()
```

Comment:
Move up to `s.Start()` and use `defer` to ensure execution regardless of code path.

Reply:
Done with commit 84de7f0c9c6094ee525ad5b8033ae42f11225d03.